src/components/OTP.vue

```diff
@@ -241,6 +314,11 @@ export default {
         : baseStyle + "pointer-events-none opacity-30";
     },
 
+    /** Starts the timer */
+    startTimer() {
+      this.interval = setInterval(() => (this.resendOTPTimer -= 1), 1000);
```

Comment:
I can't see this `interval` defined anywhere
I can only see `OTPInterval` defined and that is not being used anywhere
Did you mean to use `OTPInterval` to store the setInterval?

Reply:
Yes :|